test/src/org/omegat/core/dictionaries/LingvoDSLTest.java

```diff
@@ -0,0 +1,77 @@
+/**************************************************************************
+ OmegaT - Computer Assisted Translation (CAT) tool
+          with fuzzy matching, translation memory, keyword search,
+          glossaries, and translation leveraging into updated projects.
+
+ Copyright (C) 2015 Hiroshi Miura
+               Home page: http://www.omegat.org/
+               Support center: http://groups.yahoo.com/group/OmegaT/
+
+ This file is part of OmegaT.
+
+ OmegaT is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ OmegaT is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ **************************************************************************/
+
+package org.omegat.core.dictionaries;
+
+import java.io.File;
+import java.util.Map;
+
+import org.junit.Test;
+
+import org.omegat.core.TestCore;
+
+/**
+ * Dictionary test
+ *
+ * @author Hiroshi Miura
+ */
+public class LingvoDSLTest extends TestCore {
```

Comment:
`TestCore` includes a bunch of `org.omegat.core.Core` initialization stuff that should only be necessary for tests of GUI components. To maximize the number of tests that can be performed in a headless environment, it's best to extend `TestCase` whenever possible.